addons/l10n_id/models/res_bank.py

```diff
@@ -1,10 +1,14 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 import requests
+import pytz
 
 from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError
 
 QRIS_API_URL = "https://qris.online/restapi/qris/show_qris.php"
+QRIS_FETCH_STATUS_URL = "https://qris.online/restapi/qris/checkpaid_qris.php"
+QRIS_TIMEOUT = 30  # They say that the time to get a response vary between 6 to 30s
```

Comment:
Shouldn't we wait at least a few seconds more to be sure + handle network latency?